apps/meteor/tests/data/livechat/department.ts

```diff
@@ -59,29 +58,19 @@ new Promise((resolve, reject) => {
 
 export const createDepartmentWithAnOnlineAgent = async (): Promise<{department: ILivechatDepartment, agent: {
 	credentials: IUserCredentialsHeader;
-	user: IUser;
+	user: IUser & { username: string };
```

Comment:
What about adding a type like this (in case we don't have it)
```
type WithRequiredProperty<Type, Key extends keyof Type> = Type & {
  [Property in Key]-?: Type[Property];
};

WithRequiredProperty<IUser, 'username'>
```

Reply:
Yups nice! Much better :ok_hand: